crypto/bn/bn_print.c

```diff
@@ -138,7 +138,7 @@ int BN_hex2bn(BIGNUM **bn, const char *a)
         a++;
     }
 
-    for (i = 0; i <= (INT_MAX/4) && isxdigit((unsigned char)a[i]); i++)
+    for (i = 0; i <= (INT_MAX/4) && ossl_isxdigit((unsigned char)a[i]); i++)
```

Comment:
So, for this function `BN_hex2bn()`, is the input argument a localized string (i.e. either ASCII or EBCDIC) or is it always ASCII? For EBCDIC platforms, maybe it can auto-detect based on the first character? (*a > 0x7F, it's EBCDIC, otherwise, assume ASCII input)